spec/features/syllabus_upload_spec.rb

```diff
@@ -28,7 +28,8 @@
       visit "/courses/#{course.slug}?syllabus_upload=true"
       expect(page).to have_content course.title
       expect(page).to have_content 'Syllabus'
-      click_button 'edit'
+      expect(page).to have_selector('.module.course-description.course__syllabus-upload__inner')
+      click_button 'edit' if page.has_button?('edit')
```

Comment:
Can you link the CI log from where this spec failed? It's not clear to me why this change will help make it more reliable.

Reply:
Here is the build
https://github.com/WikiEducationFoundation/WikiEduDashboard/actions/runs/11393351404/job/31969011021